src/domain_factory/account/depopt_claim_account.py

```diff
@@ -0,0 +1,28 @@
+from domain_factory.account.external_counterparty_account import ECPAccount
+
+
+class DepotClaimAccount(ECPAccount):
```

Comment:
Is there a typo in the name of this file?

Reply:
done